ratelimits/limit.go

```diff
@@ -157,21 +161,24 @@ func loadAndParseOverrideLimits(path string) (limits, error) {
 				return nil, fmt.Errorf("unrecognized name %q in override limit, must be one of %v", k, limitNames)
 			}
 			v.limit.name = name
-			v.limit.isOverride = true
+
 			for _, entry := range v.Ids {
+				limit := v.limit
```

Comment:
Since v.limit doesn't vary from loop to loop, it would make more sense to do this assignment outside the loop. If you move it all the way to the top of the containing loop (L155), you can also replace some other instances of `v.limit` with just `limit` (which I believe is the purpose of the assignment, to create a shorthand).

Reply:
The purpose of this assignment is to create a copy-by-value of the limit, so that we can set fields which _do_ vary per-loop (namely the overrideKey) without affecting the v.limit object from the parent loop.

Reply:
This was one of those things that didn't _need_ to be fixed but could certainly lead to some unpleasantness if someone were to refactor it at a later date. So we decided to fix it now.